go.mod

```diff
@@ -2,37 +2,34 @@ module github.com/hyperledger-labs/fabric-private-chaincode
 
 go 1.14
 
+// Note:
+// - fabric has a go.mod but the normal tagging, e.g., v2.2.0 does NOT
+//   follow go module versioning, where API with version > v1 have to be explicitly
+//   versioned.
+//   the workaround is to update the module not based on version tag, e.g.,
+//      go get github.com/hyperledger/fabric@v2.2.0
+//   (which will fail) but using the commit id or a branch name
+//      go get github.com/hyperledger/fabric@release-2.2
+//   The version attributed to, though, seems rather random but, oh, well, ....
+// - furthermore, try to keep versions here as much as possible in sync
+//   and go mod tidy'ed as additional or newer dependencies can pull in
+//   versions which make fabric tools, e.g., configtxgen, fail mysteriously
+//   at runtime. (Note though keeping them identical in version will often
+//   not be possible ....)
+
 require (
-	github.com/Knetic/govaluate v3.0.0+incompatible // indirect
-	github.com/Shopify/sarama v1.26.1 // indirect
-	github.com/VictoriaMetrics/fastcache v1.5.7 // indirect
 	github.com/dustin/go-broadcast v0.0.0-20171205050544-f664265f5a66
-	github.com/fsouza/go-dockerclient v1.6.5 // indirect
 	github.com/gin-contrib/cors v1.3.1
 	github.com/gin-gonic/gin v1.6.1
 	github.com/golang/protobuf v1.3.5
-	github.com/hashicorp/go-version v1.2.0 // indirect
-	github.com/hyperledger-labs/fabric-private-chaincode/ercc v0.0.0-20200323171133-70f103dd66ee
-	github.com/hyperledger/fabric v2.1.1+incompatible
-	github.com/hyperledger/fabric-amcl v0.0.0-20200424173818-327c9e2cf77a // indirect
+	github.com/hyperledger-labs/fabric-private-chaincode/ercc v0.0.0-20200708113644-adeb2609f5e7
+	github.com/hyperledger/fabric v1.4.0-rc1.0.20200715015833-3741860ac90f
```

Comment:
v1.4.0?

Reply:
so comment on top of file ....  (fabric api versioning is not consistent with go module conventions and so you cannot use v2.2.0, but the comment corresponds to the same)